src/index.js

```diff
@@ -4,16 +4,17 @@ import { Provider } from 'react-redux'
 import { createStore } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import { rootReducer } from './reducers'
+import { HashRouter as Router} from 'react-router-dom';
```

Comment:
We went with HashRouter over BrowserRouter to allow deployment to be a smoother process, but we should probably research the differences between the two.